test/integration/util/util.go

```diff
@@ -72,6 +72,24 @@ func StartScheduler(clientSet clientset.Interface) (factory.Configurator, Shutdo
 	}
 
 	sched := scheduler.NewFromConfig(config)
+	scheduler.AddAllEventHandlers(sched,
+		v1.DefaultSchedulerName,
+		informerFactory.Core().V1().Nodes(),
+		informerFactory.Core().V1().Pods(),
+		informerFactory.Core().V1().PersistentVolumes(),
+		informerFactory.Core().V1().PersistentVolumeClaims(),
+		informerFactory.Core().V1().ReplicationControllers(),
+		informerFactory.Apps().V1().ReplicaSets(),
+		informerFactory.Apps().V1().StatefulSets(),
+		informerFactory.Core().V1().Services(),
+		informerFactory.Policy().V1beta1().PodDisruptionBudgets(),
+		informerFactory.Storage().V1().StorageClasses(),
+	)
+
+	go func() {
```

Comment:
I think this goroutine is redundant with the one created in factory.NewConfigFactory